docs/conf.py

```diff
@@ -325,7 +325,9 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None)
```

Comment:
@hachreak we should change the assert to ValueError in `invenio_pages.models:78`.